### PR TITLE
provisioner: Add ability to set kernel parameters to discovery image

### PR DESCRIPTION
**Pull request by vuntz**
There's a new discovery.append attribute in the proposal.

### CODE REVIEW

**Comment by jsuchome** [comment]
+1

File: chef/data_bags/crowbar/migrate/provisioner/035_discovery_append.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def upgrade(ta, td, a, d)
+  a["discovery"] = ta["discovery"]
+
+  return a, d
+end
+
+def downgrade(ta, td, a, d)
+  a["discovery"] = ta["discovery"]
```
Comment (toabctl) [inline chef/data_bags/crowbar/migrate/provisioner/035_discovery_append.rb]: Shouldn't you remove the whole ```discovery``` key here?

Comment (tboerger) [inline chef/data_bags/crowbar/migrate/provisioner/035_discovery_append.rb]: IMHO yes

Comment (vuntz) [inline chef/data_bags/crowbar/migrate/provisioner/035_discovery_append.rb]: Sorry, copy and paste mistake. Fixed.

**Comment by toabctl** [comment]
+1

**Pull request merged**